prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -365,7 +389,14 @@ class Hl7Serializer(
                 itemIndex = messageIndex
             )
         } catch (e: Exception) {
-            val msg = "${e.localizedMessage} ${e.stackTraceToString()}"
+            val msg = when (e) {
+                // Couldn't use InvalidPhoneMessage because it requires params we
+                // cannot access from this scope
+                is NumberParseException ->
+                    "The string supplied is not a valid phone number. Reformat to a 10-digit phone number (e.g. (555)" +
+                        " 555-5555)."
```

Comment:
This is not the only format we accept for phone numbers. We also accept international numbers if they contain the country code. Your message should reflect that.

Reply:
Please refer to the pre-approved error messages work done in tandem between Seth and the O&O team: https://cdc.sharepoint.com/:x:/r/teams/USDSatCDC/_layouts/15/Doc.aspx?sourcedoc=%7B3fac1cd7-3045-479b-b46e-1f2550a3415e%7D&action=editnew&cid=598df61b-8bdd-4e59-ac83-2bae83b41783

Reply:
Update: From O&O sync on Wed, Oct 5

We will forego updating this error message entirely for this ticket, and wait for approved messaging on the full details of telephone number support! I will remove this line and the error message will remain as follows for now:

> The string supplied did not seem to be a phone number. Error type: NOT_A_NUMBER. The string supplied did not seem to be a phone number. at com.google.i18n.phonen